napari/plugins/_tests/test_sample_data.py

```diff
@@ -26,6 +26,8 @@ def test_sample_hook(test_plugin_manager, monkeypatch):
 
     with pytest.raises(KeyError) as e:
         viewer.open_sample('test_plugin', 'random data')
+
+        print([str(e)])
```

Comment:
debug ?

Reply:
Well it depends... when this was failing having the print provides a way to actually fix it, so that is why I added it :)